apps/search/style/search.css

```diff
@@ -222,6 +222,7 @@ section.apps {
   /* Hide the whitespace between inline elements */
   font-size: 0;
   height: 10rem;
+  width: 32rem;
```

Comment:
Nope, the point was in landscape mode we want to restrict the width to the same as portrait mode so the icons are displayed at the same size

Ultimitely I think we want to have it 100% and horizontal scroll to the results, but I think thats a reasonably large patch so this one restricts us to 4 results that are displayed the same horizontal or vertical